packages/stockflux-launcher/src/app-shortcuts/AppShortcuts.js

```diff
@@ -25,26 +25,11 @@ export default () => {
       {apps
         .filter(
           app =>
-            app.customConfig !== undefined &&
-            app.customConfig.showInLauncher &&
-            app.appId.indexOf("stockflux-") === 0
```

Comment:
Something has gone wrong here. Your PR which refactors the app shortcuts seems to have disappeared. I noticed because the check I added here to ensure that the `appId` begins with `stockflux-` which I added into your PR has been deleted.

Reply:
Yeah, I accidentally chose wrong changes when resolving conflicts. Should be fixed now, thanks for noticing.